vuepress/docs/next/docs/curate/bundle-versions-faq.md

```diff
@@ -1,77 +1,58 @@
 
-## What format should I use to version my bundle?
+Micro frontends and microservices can have their own version numbers, independent of the bundle version, and can be updated in the same way.
 
-Entando requires versions to follow the [semantic versioning 2.0.0](https://semver.org/#semantic-versioning-200), with the possibility to prepend a `v` to the version itself. Some valid bundle versions are:
+### 2. My bundle contains a microservice generated with the Entando Component Generator, does the version of the microservice have to be the same as the bundle version?
 
-- 1.0
-- v0.1.0-alpha
-- 1.0.1-SNAPSHOT
-- 22.109.10234-RC.1
+The version of the microservice - or the Docker image associated with the microservice - isn't bound to the version of the bundle containing the microservice itself. 
 
+This gives the bundle developer complete control over the bundle release process, especially in those situations where the bundle contains more components and even more microservices.
 
-## As a bundle developer, how should I create a new version of a bundle?
-If you're a bundle developer and you want to release a new version of your bundle, you can simply update the content of your bundle and commit the changes to the bundle Git repository. 
-Once all the required changes are committed, you can create a new tag in the Git repository using the `git tag` command and publish it to the remote repository using the `git push --tags` command
 
-For example, let's assume my bundle already has a version `1.0.0` and I want to publish version `2.0.0` of my bundle; here are the commands I need to use to create and publish the new tag:
+## Conventions 
+ 
+### 1. How is a bundle version defined?
 
-```
-    git tag -a "2.0.0" -m "My new version"
-    git push --tags
-```
+Bundle versions are defined by the creator and set in the bundle descriptor entando.json. You can have multiple versions of a bundle as long as they are defined and tagged as such.
+ 
+```json
+{
+  "name": "my-bundle",
+  "description": "This is a description of my-bundle",
+  "type" : "bundle",
+  "version": "1.0.0",
+  "svc": [...]
+  ...
 
-## How to prevent a particular bundle version from being visible in the Local Hub?
+}
+```
+### 2. What format should I use to version my bundle?
 
-If, for some reason, you don't want a particular version to be available for the installation, you can proceed as follows:
+Follow the recommended [semantic versioning 2.0.0](https://semver.org/#semantic-versioning-200), with the possibility to prepend a `v` to the version itself. Some valid bundle versions are:
 
-1. You can edit the bundle directly using the command `kubectl edit entandodebundles.entando.org -n <namespace-of-the-bundle> <name-of-the-bundle>`
+- 1.0
+- v0.1.0-alpha
+- 1.0.1-SNAPSHOT
```

Comment:
Remove line 49 and 50. Those are not valid in the current semantic versioning we use which is only x.y.z and vx,y,z,